apps/client/src/features/app-settings/panel/manage-panel/ManageRundowns.tsx

```diff
@@ -119,6 +165,17 @@ export default function ManageRundowns() {
                             },
                             {
                               type: 'item',
+                              icon: IoDuplicateOutline,
+                              label: 'Duplicate',
+                              onClick: () => {
+                                setActionError(null);
+                                setTargetRundown(id);
```

Comment:
`useState` has not updated `targetRundown` by the time `duplicate` is called so the id is stale and refers to the previously used id